db/migrate/20150513121721_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Good to know this. 
Actually, after implementing token resetting on password reset, i changed to the more secure solution: token creation before (and token deletion after) password-reset. So the above migration is not necessary any more. Since that is the last migration, can i just roll it back and `git rm` the migration file? Although not advisable generally for a migration, in this case since it doesn't affect the schema, i am inclined to use it. What do you think?